packages/contract/lib/execute.js

```diff
@@ -183,7 +187,7 @@ const execute = {
       const promiEvent = new PromiEvent(false, constructor.debugger);
 
       execute
-        .prepareCall(constructor, methodABI, arguments)
+        .prepareCall(constructor, methodABI, arguments, false)
```

Comment:
Don't even pass `false`, just leave the fourth argument off. Then in `prepareCall` you can do something 
```
if (isCall) {
  return { args, params };
}
const network = await constructor...
```
You know what I mean?